shared-src/net/wurstclient/features/mods/other/MurderMysteryMod.java

```diff
@@ -122,39 +123,46 @@ public void onUpdate()
 		updateMS();
 		if(!hasTimePassedM(250))
 			return;
-		whoHasSword();
+		if(amINOTTheMurderer())
+			whoHasSword();
 		updateLastMS();
 	}
 
-	private void whoHasSword()
+	private boolean amINOTTheMurderer()
```

Comment:
@DeathsGun @liach Is changing it to positive worth the small impact on performance?